WordPress/Classes/ViewRelated/Jetpack/Jetpack Scan/View Models/JetpackScanThreatViewModel.swift

```diff
@@ -309,8 +311,128 @@ struct JetpackScanThreatViewModel {
+                                             withPad: Constants.noBreakSpace, startingAt: 0)
+
+            let padding = " "
+            let contentsStr = String(format: "%@\n", padding + contents + contentsPadding)
+            let contentsAttr = NSMutableAttributedString(string: contentsStr, attributes: [
+                NSAttributedString.Key.font: lineFont,
+                NSAttributedString.Key.foregroundColor: Constants.normal.textColor,
+                NSAttributedString.Key.backgroundColor: Constants.normal.backgroundColor,
+                NSAttributedString.Key.paragraphStyle: paragraph,
+            ])
+
+            var numberBackgroundColor = Constants.normal.numberBackgroundColor
+
+            // Highlight logic
+
+            if let highlights = line.highlights {
+
+                numberBackgroundColor = Constants.highlighted.numberBackgroundColor
+
+                contentsAttr.addAttribute(.backgroundColor,
+                                          value: numberBackgroundColor,
+                                          range: NSRange(location: 0, length: contentsAttr.length))
+
+                for highlight in highlights {
+                    let location = highlight.location
+                    let length = highlight.length + padding.count
+                    let range = NSRange(location: location, length: length)
+
+                    contentsAttr.addAttributes([
+                        NSAttributedString.Key.font: lineFont,
+                        NSAttributedString.Key.foregroundColor: Constants.highlighted.textColor,
+                        NSAttributedString.Key.backgroundColor: Constants.highlighted.backgroundColor,
+                        NSAttributedString.Key.paragraphStyle: paragraph
+                    ], range: range)
+                }
+            }
+
+            numberAttr.setAttributes([
+                NSAttributedString.Key.font: numberFont,
+                NSAttributedString.Key.foregroundColor: Constants.normal.numberTextColor,
+                NSAttributedString.Key.backgroundColor: numberBackgroundColor,
+            ], range: NSRange(location: 0, length: numberStr.count))
+
+            attrString.append(numberAttr)
+            attrString.append(NSAttributedString(string: ""))
+            attrString.append(contentsAttr)
+        }
+
+        return attrString
+    }
+
+    private struct Constants {
+
+        static let noBreakSpace = "\u{00a0}"
+
+        struct normal {
+            static let textColor = UIColor(rgb: 0x2C3337)
+            static let backgroundColor = UIColor(rgb: 0xF6F7F7)
+            static let numberTextColor = UIColor(rgb: 0x646970)
+            static let numberBackgroundColor = UIColor(rgb: 0xC3C4C7)
+        }
+
+        struct highlighted {
+            static let textColor = UIColor.white
+            static let backgroundColor = UIColor(rgb: 0xD63638)
+            static let numberBackgroundColor = UIColor(rgb: 0xF1DADA)
```

Comment:
I think we can just convert these to normal UIColors so we don't need to add the hex to UIColor extension below. 

I normally use https://www.uicolor.io/ to help with this.